packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/SubjectGroupViewer/SubjectGroupViewer.js

```diff
@@ -21,6 +27,29 @@ const Container = styled.div`
   }
 `
 
+/*
+Note on Subject Viewer sizing/fitting:
+- The grid should fit the height OR width of the available visible space.
+- This is implemented with the container div AND the <svg> having
+  width/height=100% (which 'fits') an optional max-width/height (which
+  'restricts').
+- Curious note: the max-width/height has to be repeated in <svg> due to
+  Safari 12. If we ignore Safari, we only need max-width/height on the
+  container div, not the <svg>
+ */
+const SVG = styled.svg`
```

Comment:
```suggestion
export const SVG = styled.svg`
```

See comment on line 51...